### PR TITLE
Reduce dependency on python2

### PR DESCRIPTION
**Pull request by Zopolis4**
Working on #7934. 

### Run the following to get this pull request's changes locally for testing.
```bash
CREW_REPO=https://github.com/Zopolis4/chromebrew.git CREW_BRANCH=ske crew update
```

### CODE REVIEW

**Comment by satmandu** [comment]
Thanks for working on this!

File: packages/py3_dool.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require 'buildsystems/pip'
```
Comment (satmandu) [inline packages/py3_dool.rb:1]: Please add this to tools/packages.yaml?

**Review by satmandu — APPROVED** [review]

File: packages/py3_dool.rb (hunk 1 above)
Comment (Zopolis4) [reply packages/py3_dool.rb:1]: Added.

File: packages/ecasound.rb [hunk 2]
```diff
@@ -1,41 +1,42 @@
...
+  depends_on 'libvorbis' # R
+  depends_on 'ncurses' # R
+  depends_on 'opus' # R
+  depends_on 'readline'
+  depends_on 'ruby'
 
-  def self.install
-    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
-  end
+  configure_options '--disable-pyecasound'
```
Comment (uberhacker) [inline packages/ecasound.rb:40]: What happened to the manpages?  It's fine to leave out documentation but manpages should be installed, if possible.

File: packages/dropbox.rb [hunk 3]
```diff
@@ -1,59 +1,30 @@
-require 'package'
+require 'buildsystems/autotools'
 
-class Dropbox < Package
-  description 'Dropbox simplifies the way you create, share and collaborate.  Bring your photos, docs, and videos anywhere and keep your files safe.'
+class Dropbox < Autotools
+  description 'Dropbox Integration for Nautilus'
```
Comment (uberhacker) [inline packages/dropbox.rb:4]: Why the description change?  Dropbox is not just for Nautilus.

File: packages/dropbox.rb [hunk 4]
```diff
@@ -1,59 +1,30 @@
...
-    source_sha256 '3affb0ade20e26adf337489c6b780a94ddd8d51712a3bd88be2def38b5689570'
-  when 'x86_64'
-    source_url 'https://clientupdates.dropboxstatic.com/dbx-releng/client/dropbox-lnx.x86_64-48.4.58.tar.gz'
-    source_sha256 'bd948a712e21c36d01ad17e72a5893972dd9411408561c3b806dbca2c32ae9a4'
-  end
-  binary_compression 'tar.xz'
+  version '2024.04.17'
+  license 'GPL3+'
+  compatibility 'aarch64 armv7l x86_64'
+  source_url 'https://github.com/dropbox/nautilus-dropbox'
```
Comment (uberhacker) [inline packages/dropbox.rb:9]: OK, so this is confusing.  The package name should be `Nautilus_dropbox`.  Dropbox is a legitimate app we should not just rename like this.

Comment (uberhacker) [reply packages/dropbox.rb:9]: EDIT: After a bit of research, I now understand how this should be installed on Linux as explained [here](https://help.dropbox.com/installs/linux-commands#How-to-build-the-Dropbox-installer-for-Linux-from-source).  Disregard my earlier comment.

File: packages/dropbox.rb (hunk 3 above)
Comment (uberhacker) [reply packages/dropbox.rb:4]: I think we should leave the description unchanged but add integration for Nautilus.

File: packages/ecasound.rb (hunk 2 above)
Comment (Zopolis4) [reply packages/ecasound.rb:40]: No idea, I wasn't able to get them to work.

**Review by satmandu — APPROVED** [review]

File: packages/ecasound.rb (hunk 2 above)
Comment (satmandu) [reply packages/ecasound.rb:40]: New version of a package... people can go to the web site to get docs...

**Pull request merged**